TestingApp/ViewController/ViewController.swift

```diff
@@ -412,7 +412,7 @@ extension ViewController {
             title: "Create Authentication",
             style: .default
         ) { _ in
-            authentication.authenticate(with: enteredJwt, accessToken: enteredAccessToken) { [weak self] result in
+            authentication.authenticate(with: enteredJwt, accessToken: enteredAccessToken.isEmpty ? nil : enteredAccessToken) { [weak self] result in
```

Comment:
I know that we are a little bit in a hurry but this line is enormous :) can the access token be its own variable and then used here?

Reply:
fixed - cd1dcd461e9552867685c3d2f28e35dfb0e986fd